downloader.py

```diff
@@ -31,13 +31,13 @@ def main(argv):
 	email = ''
 	password = ''
 	directory = 'packtpub_media'
-	formats = 'pdf,mobi,epub,jpg'
+	fileTypes = 'pdf,mobi,epub,jpg,code'
 	includeCode = False
```

Comment:
You can remove includeCode variable as using other approach for downloading code.